docs/madr/decisions/039-transition-to-new-policies.md

```diff
@@ -0,0 +1,642 @@
+# Transition to new policies
+
+- Status: accepted
+
+## Context and Problem Statement
+
+The current state of the policies as of Kuma 2.6.x:
+* new policies are in GA, they're the recommended way to configure new deployments in Kuma
+* old policies are not deprecated, they're still in use by majority of Kuma users
+* as long as there is at least 1 new policy a type, the old policies of the same type are not applied
+  (i.e. if there is a MeshTimeout policy, the Timeout policies is not applied)
+
+The goal is to provide a way to transition from old policies to new policies. Having a smooth transition is going to enable 
+us to deprecate old policies in the future.
+
+Prerequisites for the transition:
+* both Global and Zone CPs are running the latest Kuma version
+
+## Decision Drivers
+
+- Predictability: when replacement policies are created (manually or automatically), user should be able to verify them 
+without applying for real
+- Short feedback loop: we'd like to deliver small portions of new policies more often, rather than big portions but rarely
+- Quick rollback: user should be able to rollback to old policies
+
+## Considered Options
+
+- Introduce a shadow mode for policies on Zone CP ✅
+- Introduce a shadow mode for policies on Global CP
+  - Show potential changes for a specific DPP
+  - Show potential changes for all affected DPPs
+
+## Shadow mode for policies
+
+Shadow mode is a way to test new policies without affecting the real traffic. When the policy is created with `kuma.io/effect: shadow` label
+
+```yaml
+type: MeshTimeout
+name: timeout-global
+mesh: default
+labels:
+  kuma.io/effect: shadow
+spec:
+  targetRef:
+    kind: Mesh
+  to: [...]
+```
+
+it doesn't apply on the real proxies. Instead, by using Inspect API we can inspect what config dump would look like if the shadow policies was applied.
+
+### Changes in the Inspect API
+
+Introduce a new non-physical endpoint `/_snapshot` that returns a config dump of a DPP without the actual `/config_dump` call to Envoy.
```

Comment:
what is non-physical?

Reply:
I was using the terminology from Charly's [comment](https://github.com/kumahq/kuma/pull/9571#discussion_r1521620338):
> Non physical endpoints are usually prefixed by _ like _rules or _resources.

IIUC it's non-physical when there is no such resource type

Reply:
yeah "non physical" would be this. It's a very crap naming though :)